tests/integrationv2/test_happy_path.py

```diff
@@ -57,7 +58,14 @@ def test_s2n_server_happy_path(managed_process, cipher, provider, curve, protoco
     # the stdout reliably.
     for results in server.get_results():
         assert results.exception is None
-        assert results.exit_code == 0
+        '''
+        In TLS1.3 the JDK sends a user_canceled alert as well as a close_notify alert
+        after a connection closes, so s2n will register the alert and complain.
```

Comment:
How does the alert messages get translated in s2n, do we have a specific error code for it?